capstone/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -31,6 +35,8 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     UserService userService = UserServiceFactory.getUserService();
     if (userService.isUserLoggedIn()) {
       userId = userService.getCurrentUser().getUserId();
+    } else {
+      errorHandler.sendError(response, "User is not logged in.");
     }
```

Comment:
Lucy made an AuthenticatedServlet that you can extend instead of the HttpServlet that will do the user Authentication for you: https://github.com/googleinterns/step100-2020/blob/master/capstone/src/main/java/com/google/sps/servlets/AuthenticatedServlet.java
So you just need to make sure to extend the AuthenticatedServlet and you can remove an user login related code. Also note the doPost/doGet change to give you access to the userID

Reply:
Sounds good, added to my servlets